CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * Added a new conditional configuration `--when.platforms` to include
   settings only on certain platforms.
 
+* `jj git remote` family of commands now supports different fetch and push URLs.
```

Comment:
nit: Can you squash the last commit? We usually include doc/test changes in the implementation commits.